### PR TITLE
feat: Devtron feedback impl

### PR DESCRIPTION
**Pull request by shivani170**
# Description

To add the `Give Feedback` form in Help section
Screenshot 2023-02-02 at 4 46 27 PM

Screenshot 2023-02-02 at 4 47 08 PM

Fixes [#AB1885](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/Devtron%20Team/Stories/?workitem=1885)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- **Go to the Help Nav > Click on Give feedback It should open the  Typeform**
- **After click on cross Help nav should closs**

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/common/navigation/navigation.scss [hunk 1]
```diff
@@ -256,6 +256,10 @@
     left: 68px;
 }
 
+.help-grid__feedback {
+    grid-template-rows: 36px 36px 1px 36px 36px 1px 36px 36px auto !important;
```
Comment (sohel-rp) [inline src/components/common/navigation/navigation.scss:260]: Do we need `!important` here?

File: src/components/common/header/HelpNav.tsx [hunk 2]
```diff
@@ -87,6 +98,18 @@ function HelpNav({ className, showHelpCard, setShowHelpCard, serverInfo, fetchin
                         </Fragment>
                     )
                 })}
+                {isEnterprise && (
+                    <div onClick={stopPropagation} className="help-card__option help-card__link flex left cn-9">
+                        <Feedback />
+                        <SliderButton
+                            className="dc__transparent help-card__option-name ml-12 cn-9 fs-14"
+                            id={`UheGN3KJ#source=${window.location.hostname}`}
```
Comment (sohel-rp) [inline src/components/common/header/HelpNav.tsx:106]: can we keep it in some const?

File: src/components/common/header/HelpNav.tsx [hunk 3]
```diff
@@ -48,12 +52,23 @@ function HelpNav({ className, showHelpCard, setShowHelpCard, serverInfo, fetchin
     ]
 
     const onClickGettingStarted = () => {
-      setGettingStartedClicked(true)
+        setGettingStartedClicked(true)
+    }
+
+    const onClickHelpOptions = (option: HelpOptionType): void => {
+        ReactGA.event({
+            category: 'Main Navigation',
```
Comment (sohel-rp) [inline src/components/common/header/HelpNav.tsx:60]: Should it be Main Navigation?

File: src/components/common/header/HelpNav.tsx [hunk 4]
```diff
@@ -32,7 +36,7 @@ function HelpNav({ className, showHelpCard, setShowHelpCard, serverInfo, fetchin
             name: 'Chat with support',
             link: 'https://discord.devtron.ai/',
             icon: Chat,
-            showSeparator: true,
+            showSeparator: isEnterprise ? false : true,
```
Comment (sohel-rp) [inline src/components/common/header/HelpNav.tsx:39]: can be simplified by using `!isEnterprise` directly

File: src/components/common/navigation/navigation.scss (hunk 1 above)
Comment (shivani170) [reply src/components/common/navigation/navigation.scss:260]: Yes as .help-card will override this one

File: src/components/common/header/HelpNav.tsx (hunk 2 above)
Comment (shivani170) [reply src/components/common/header/HelpNav.tsx]: Done

File: src/components/common/header/HelpNav.tsx (hunk 4 above)
Comment (shivani170) [reply src/components/common/header/HelpNav.tsx]: Done

**Review by milindk8 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Pull request merged**